src/ray/rpc/object_manager/object_manager_client.h

```diff
@@ -48,19 +42,17 @@ class ObjectManagerClient {
   /// \param request The request message.
   /// \param callback The callback function that handles reply from server
   void Push(const PushRequest &request, const ClientCallback<PushReply> &callback) {
-    client_call_manager_.CreateCall<ObjectManagerService, PushRequest, PushReply>(
-        *stubs_[push_rr_index_++ % num_connections_],
-        &ObjectManagerService::Stub::PrepareAsyncPush, request, callback);
+    RPC_CALL_METHOD_WITH_CLIENT(ObjectManagerService, Push, request, callback,
+                                rpc_clients_[push_rr_index_++ % num_connections_]);
```

Comment:
I know it's not introduced by your PR but mind adding a comment about what this magical indexing does?

Reply:
sure.